autorest/azure/azure.go

```diff
@@ -50,8 +51,8 @@ type RequestError struct {
 
 // Error returns a human-friendly error message from service error.
 func (e RequestError) Error() string {
-	return fmt.Sprintf("azure: Service returned an error. Code=%q Message=%q Status=%d",
-		e.ServiceError.Code, e.ServiceError.Message, e.StatusCode)
+	return fmt.Sprintf("autorest/azure: Service returned an error. Code=%q Message=%q Details=%v Status=%v",
+		e.ServiceError.Code, e.ServiceError.Message, e.ServiceError.Details, e.StatusCode)
```

Comment:
@garimakhulbe if it is wrapped in there, you should probably just implement `Error()` on this, and instead of reaching its members, perhaps you should just do %v on the err variable, which would print the Error() output.